### DISCUSSION TITLE
Connect to LMU64

### DISCUSSION THREAD

**Discussion by akula3** [fredlcore/BSB-LAN#484] [Q&A] [answered]
How to connect? Just instead of RRG considering +/- ?

**Comment by fredlcore** [comment]
Read the manual first, and if you still have questions, please be more specific. To connect to LMU64, you need a OCI420 in order to connect via LPB. The room unit uses OpenTherm which is completely different from BSB/LPB.

**Comment by akula3** [comment]
I red the manual. And I found a description only for upgrade the boiler with OCI420 to be able to comminicate via LPB protocol.
I do not want to invest in OCI420, but want to use the PPS protocol. It seems, it is supported for LMU64:
![grafik](https://user-images.githubusercontent.com/58461189/190079862-e426130f-6f17-48b8-bd47-37a9e8f003a0.png)
I found in the manual also example pictures for different controllers, but not for LMU64, just this description:
![grafik](https://user-images.githubusercontent.com/58461189/190081247-30705924-9d1b-4225-9472-45585bf65c4a.png)
But I can't find the mentioned pins on my LMU64.

**Reply by 1coderookie** [reply, marked as answer]
Thanks for that information, that is an error in the description, the LMU64 does NOT have a PPS! I just fixed it.
As you can see in the special chapter about LMU64
https://1coderookie.github.io/BSB-LPB-LAN_EN/chap10.html#1024-note-special-case-lmu54lmu64-controllers
this controller needs an OCI420 to retrofit LPB for being able to use it with the BSB-LAN adapter.
